packages/gatsby-plugin-gatsby-cloud/src/__tests__/gatsby-browser.js

```diff
@@ -210,6 +210,13 @@ describe(`Preview status indicator`, () => {
   //   })
   // })
 
+  /**
+   * SKIPPED TEST NOTE
+   * Some tests are skipped and this is because they were not working as expected and
```

Comment:
Please clarify here and/or in the PR description:

- What "not working as expected" means
- What changes would be necessary to fix this
- How bad skipping these tests is for coverage
- If they can't be fixed in this test, when the follow-up PR would be put up

Reply:
Add more detail to the not let me know if more is needed.